src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationILGen.cs

```diff
@@ -65,6 +65,15 @@ internal ModuleBuilder ModuleBuilder
         }
         internal TypeAttributes TypeAttributes { get { return _typeAttributes; } }
 
+        private static readonly string[] s_type = new string[] { "type" };
+        private static readonly string[] s_xmlReader = new string[] { "xmlReader" };
+        private static readonly string[] s_objectToSerializeWriter = new string[] { "objectToSerialize", "writer" };
+        private static readonly string[] s_reader = new string[] { "reader" };
+        private static readonly Type[] s_aType = new Type[] { typeof(Type) };
```

Comment:
I think it's confusing that both `s_checkType` and `s_reader` are array of strings and `s_objectType` is an array of Types and here we have `s_aType` vs `s_type` ...

Suggest making things consistent in these files, eg `s_XXXType` is an array of Types, and `s_XXXString` is an arry of strings. Eg., `s_objectType`   `s_xmlReaderString`  `s_checkTypeStrings`   `s_objectAndSerializeWriterStrings` `s_typeString` `s_readerType` where it's singular for one thing in the array and plural for more.

that ought to allow me to understand what a thing is without going to look.

Reply:
Done